src/Explorer/QueryCopilot/QueryCopilotPromptbar.tsx

```diff
@@ -318,6 +321,46 @@ export const QueryCopilotPromptbar: React.FC<QueryCopilotPromptProps> = ({
     useTabs.getState().setIsQueryErrorThrown(false);
   }, []);
 
+  const suggestionsnhistory = filteredHistories.concat(filteredSuggestedPrompts.map((item) => item.text));
+
+  const handlekeydown: React.KeyboardEventHandler = (e) => {
+    const { key } = e;
+    let nexIndexCount = 0;
+    if (key === "ArrowDown") {
```

Comment:
Should this use a constant like in pr 1472?

Reply:
It has been changed to const instead of let